src/algo/__test__/routing.test.js

```diff
@@ -26,7 +26,10 @@ describe('Call route optimization endpoints', () => {
         statusCode: 400,
         statusText: 'Bad Request',
         errorMessage: [
-          { key: '0', messages: 'Route Setting does not belong to you' },
+          {
+            key: 'route_setting_id',
+            messages: ['Route Setting does not belong to you'],
```

Comment:
snakecase or camel, you will need to choose @vinay3206

Reply:
updated keys to camelCase